cmd/snap-repair/runner.go

```diff
@@ -49,6 +49,7 @@ import (
 	"github.com/snapcore/snapd/httputil"
 	"github.com/snapcore/snapd/logger"
 	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/overlord/configstate/configcore"
```

Comment:
in general configcore is designed to be built using build tags such that the fs-only subset of its functionality can be imported in most tools outside of snapd, while in general we don't want to import overlord manager bits in anything but snapd itself. Is still the case that usually we try to define APIs shared between configcore and the rest of the tools in sysconfig (you can take a look there), so something like RepairConfig would live in sysconfig. OTOH snap-repair is very special, as the fallback mechanism if snapd itself cannot operate we try to keep its import to a minimum, so for now would we would likely do is repeat the very small definition of an unexported repairConfig in two places. If it grows larger an approach would be to use a dedicated snap-repair/config package to share things between snap-repair and configcore.